lib/topological_inventory/orchestrator/worker.rb

```diff
@@ -100,15 +100,16 @@ def collectors_from_sources_api
               "password" => authentication["password"],
               "username" => authentication["username"],
             },
+            "tenant"          => tenant,
```

Comment:
@bdunne Why do you add tenant into digest? If it's possible to change tenant for source, then it is still the same provider with the same endpoint and credentials.
Changing of digest means that this is new source from orchestrator's/openshift's point of view => which means kill and redeploy. 
And this is not necessary.

Reply:
I need it to update the source record [here](https://github.com/ManageIQ/topological_inventory-orchestrator/pull/34/files#diff-4743ee12ee7e621468f2e6590de994efR218)

Reply:
ah, ok, so you don't need it for digest but for rest api call. 
I won't keep it in #31, tenant will be saved in `Source` instance